vercel_runtime/src/lib.rs

```diff
@@ -1,19 +1,21 @@
-mod body;
 mod request;
 mod response;
+
 use lambda_runtime::LambdaEvent;
 use request::{VercelEvent, VercelRequest};
 use response::EventResponse;
 use std::future::Future;
 use tracing::{debug, error};
 
-pub type Request = lambda_http::http::Request<Body>;
+pub use lambda_http::Body;
+
 pub type Error = lambda_http::Error;
 pub type Event<'a> = LambdaEvent<VercelEvent<'a>>;
-pub use body::Body;
 pub use lambda_http::http::StatusCode;
 pub use lambda_http::service_fn;
 pub use lambda_http::tower::ServiceBuilder;
+pub use lambda_http::Request;
+pub use lambda_http::RequestExt;
```

Comment:
nit: we can probably combine a lot of these 
```
pub use lambda_http::{Request, RequestExt, Response, http::StatusCode, ...}
```

Reply:
:done: